samtranslator/model/preferences/deployment_preference_collection.py

```diff
@@ -101,9 +101,17 @@ def _codedeploy_iam_role(self):
                 }
             ],
         }
-        iam_role.ManagedPolicyArns = [
-            ArnGenerator.generate_aws_managed_policy_arn("service-role/AWSCodeDeployRoleForLambda")
-        ]
+
+        # CodeDeploy has a new managed policy. We cannot update any existing partitions, without customer reach out
+        # that support AWSCodeDeployRoleForLambda since this could regress stacks that are currently deployed.
+        if ArnGenerator.get_partition_name() in ["aws-iso", "aws-iso-b"]:
```

Comment:
We are handling a lot of strings, can't we put those in constants?

Reply:
Fair call out. I want to try and limit changes (aka fix all the mentions like this). Let me play with this and see what we can get to for setting a better pattern in the code base.

Reply:
Yes, let's merge this one this way and think about it in another PR